packages/svelte/tests/migrate/samples/single-assignment-labeled/input.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+	let count = 0;
+	let double;
+	$:{
+		double = count * 2;
+	}
+	
+	let quadruple;
+	$:{
+		quadruple = count * 4;
+		console.log("i have a side effect")
+	}
+
+	let eight_times;
+	$:{
+		// updated
+		eight_times = count * 8;
+	}
+
+	let sixteen_times;
+	$:{
+		// reassigned outside labeled statement
+		sixteen_times = count * 16;
+	}
+
+	let alot_times;
+	$:{
+		// reassigned in multiple labeled
+		alot_times = count * 32;
+	}
+	$:{
+		// reassigned in multiple labeled
+		alot_times = count * 32;
+	}
+
+	let evenmore;
+	let evenmore_doubled;
+	$:{
+		// multiple stuff in label
+		evenmore = count * 64;
+		evenmore_doubled = evenmore * 2;
+	}
+
+	let almost_infinity;
+	$: almost_infinity = count * 128;
+
+	let should_be_state;
+	$: should_be_state = 42;
```

Comment:
I also saw things like
```js
	$: should_also_be_state = 42;
```
without a preceeding `let` with the same name in the wild. Would be good to handle that aswell. and transform that into `let should_also_be_state = $state(42)`

Reply:
I planned to do that as well but since that is not a variable declaration i should've repeated a lot of code from here...i planned to do this as a separate PR to not complicate things too much in this. Should i go for this in this PR too?

Reply:
Scratch that...it was different. Pushing now.